thinc/loss.py

```diff
@@ -46,11 +46,13 @@ def __init__(
         names: Optional[List[str]] = None,
         missing_value: Optional[Union[str, int]] = None,
         neg_prefix: Optional[str] = None,
+        label_smoothing: float = 0.0,
     ):
         self.normalize = normalize
         self.names = names
         self.missing_value = missing_value
         self.neg_prefix = neg_prefix
+        self.label_smoothing = label_smoothing
```

Comment:
Would be nice to have a test for label smoothing.

Reply:
Pushed tests for `label_smoothing` in the `test_utils` for the test of `to_categorical`.